CodeGeneration/Sources/SyntaxSupport/ExprNodes.swift

```diff
@@ -66,7 +66,7 @@ public let EXPR_NODES: [Node] = [
   // arrow-expr -> 'async'? 'throws'? '->'
   // NOTE: This appears only in SequenceExpr.
   Node(
-    kind: .arrowExpr,
+    kind: .unresolvedArrowExpr,
```

Comment:
Seems like both this and `binaryOperatorExpr` stick around after folding:
```
     if let binaryOperatorExpr = op.as(BinaryOperatorExprSyntax.self) {
      return ExprSyntax(
        InfixOperatorExprSyntax(
          leftOperand: lhs,
          operatorOperand: ExprSyntax(binaryOperatorExpr),
          rightOperand: rhs
        )
      )
    }
```
and
```
    if let arrowExpr = op.as(ArrowExprSyntax.self) {
      return ExprSyntax(
        InfixOperatorExprSyntax(
          leftOperand: lhs,
          operatorOperand: ExprSyntax(arrowExpr),
          rightOperand: rhs
        )
      )
    }
```

So probably shouldn't be unresolved?

Reply:
Reverted to the non-unresolved name. Thanks for catching this.